### DISCUSSION TITLE
Show content/project type on user/author page

### DISCUSSION THREAD

**Discussion by Poopooracoocoo** [modrinth/.github#169] [Feature requests]
Currently project authors need to label it in the project summary for users to know what it is

Example:
https://modrinth.com/user/Cobblemon
![image](https://github.com/modrinth/.github/assets/51363864/d7312238-c8f5-487d-befa-aff105604300)

**Comment by triphora** [comment]
https://github.com/modrinth/.github/assets/60205699/f19b54e0-98a1-4eba-a779-64c5af9b9026

**Reply by Poopooracoocoo** [reply]
That's a filter. It doesn't allow users to see what project type a project is when they're browsing all projects by an author.

**Reply by triphora** [reply]
Is that really necessary information, though? If you care about the project type, you're going to filter; if you don't, you're not. This information used to be in the project card but was removed because it is redundant; every page that has a project card also has additional context as to what project type a given project is.

**Reply by Poopooracoocoo** [reply]
Not every author puts the project type in the project summary. I sometimes find myself browsing an author's page when I'm curious about their other projects, regardless of project type. I felt like it was necessary and that's why I created this discussion.
